### ISSUE TITLE
Redesign the planet selector component

### ISSUE THREAD

**Issue by dpora** [dpora/PlanetSite#6]
- Planet selector needs to fit the theme of the website
- Should be responsive and change for different screen sizes
- Add tabbing ability
